plugins/kapt3/kapt3-compiler/src/org/jetbrains/kotlin/kapt3/stubs/ClassFileToSourceStubConverter.kt

```diff
@@ -751,10 +751,24 @@ class ClassFileToSourceStubConverter(val kaptContext: KaptContextForStubGenerati
         val value = field.value
 
         val origin = kaptContext.origins[field]
+        // Static fields in interfaces needs an initializer, even when just processing it using APT
+        val isStaticInterfaceField = containingClass.isInterface() && isStatic(field.access)
+        val isPrimitiveOrString = Type.getType(field.desc).isOfPrimitiveOrStringType()
+        val includeInitializer = isStaticInterfaceField ||
+                !isPrimitiveOrString ||
+                when (origin?.element) {
+                    is KtProperty -> true
+                    is KtParameter -> kaptContext.options[KaptFlag.DUMP_DEFAULT_PARAMETER_VALUES] || !isFinal(field.access)
```

Comment:
Does `isFinal(field.access)` ever return true?

Reply:
Yes, I confirmed it returns `true` for private fields.